mqtt/aedes/mqtt.js

```diff
@@ -112,11 +112,13 @@ esclient.on('error', function(err) {
 function startWs() {
     var server = http.createServer();
     server.on('request', (req, res) => {
+        res.setHeader('Content-Type', 'application/json');
         if (req.url === '/version') {
             res.statusCode = 200;
-            res.setHeader('Content-Type', 'text/plain; charset=utf-8');
             res.end(`{"service":"mqtt-adapter","version":"${version}"}`);
```

Comment:
We don't use template literals (e.g mixing with ES6). Just a tip, but we should consider refactoring and using ES6.